### PR TITLE
Package calculon.0.2

### PR DESCRIPTION
**Pull request by c-cube**
### `calculon.0.2`

Library for writing IRC bots in OCaml, a collection of plugins, and a dramatic robotic actor.

The core library is called `calculon` and revolves around
the concept of commands that react to user messages. See the README for a small
tutorial on how to write your own plugins.

---
* Homepage: https://github.com/c-cube/calculon
* Source repo: https://github.com/c-cube/calculon.git
* Bug tracker: https://github.com/c-cube/calculon/issues

---

:camel: Pull-request generated by opam-publish v0.3.5

### CODE REVIEW

File: packages/calculon/calculon.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,34 @@
+opam-version: "1.2"
+maintainer: "c-cube"
+authors: ["Armael" "Enjolras" "c-cube"]
+homepage: "https://github.com/c-cube/calculon"
+bug-reports: "https://github.com/c-cube/calculon/issues"
+tags: ["irc" "bot" "factoids"]
+dev-repo: "https://github.com/c-cube/calculon.git"
+build: ["jbuilder" "build" "-p" name]
+build-test: ["jbuilder" "runtest"]
+build-doc: ["jbuilder" "build" "@doc"]
```
Comment (jpdeplaix) [inline packages/calculon/calculon.0.2/opam]: Are the two lines above not supposed to have the ```-p``` argument ? Is it fine as is ?

**Comment by jpdeplaix** [comment]
Travis fails with:
```
#=== ERROR while installing containers.2.0 ====================================#
# opam-version 1.2.2
# os           linux
# command      jbuilder runtest -p containers -j 4
# path         /home/travis/.opam/system/build/containers.2.0
# compiler     system (4.02.3)
# exit-code    1
# env-file     /home/travis/.opam/system/build/containers.2.0/containers-21574-e80101.env
# stdout-file  /home/travis/.opam/system/build/containers.2.0/containers-21574-e80101.out
# stderr-file  /home/travis/.opam/system/build/containers.2.0/containers-21574-e80101.err
### stderr ###
# File "qtest/jbuild", line 16, characters 2-228:
# Error: Library "sequence" not found.
# 
# Hint: try: jbuilder external-lib-deps --missing -p containers @runtest
```
Could you replicate https://github.com/c-cube/ocaml-containers/commit/5ebebf4fd76aa37052ed1b37dc14eea317b33c95 here as well ?

**Comment by c-cube** [comment]
This is unrelated, right?

**Comment by jpdeplaix** [comment]
@c-cube

**Comment by kit-ty-kate** [comment]
Looks good now. Thanks !

**Pull request merged**